R/carehomes.R

```diff
@@ -463,10 +523,28 @@ carehomes_parameters <- function(start_date, region,
   ret$N_tot_all <- sum(ret$N_tot)
   ret$N_tot_over25 <- sum(ret$N_tot[6:19])
   ret$N_tot_react <- sum(ret$N_tot[2:18])
+  ret$N_tot_under15 <- sum(ret$N_tot[1:3])
+  ret$N_tot_15_24 <- sum(ret$N_tot[4:5])
+  ret$N_tot_25_49 <- sum(ret$N_tot[6:10]) + (sum(ret$N_tot[18]) / 8) * 5
+  ret$N_tot_50_64 <- sum(ret$N_tot[11:13]) + (sum(ret$N_tot[18]) / 8) * 3
+  ret$N_tot_65_79 <- sum(ret$N_tot[14:16]) + sum(ret$N_tot[19]) * 0.25
+  ret$N_tot_80_plus <- sum(ret$N_tot[17]) + sum(ret$N_tot[19]) * 0.75
```

Comment:
might be useful to just add a comment explaining the way groups 18/19 are split here

Reply:
done